packages/web/src/components/gcds-search/gcds-search.css

```diff
@@ -0,0 +1,70 @@
+@layer defaults, state.focus;
+
+@layer defaults {
+  :host {
+    .gcds-search {
+      .gcds-search__header {
+        clip: rect(1px,1px,1px,1px);
+        height: 1px;
+        margin: 0;
+        overflow: hidden;
+        position: absolute;
+        width: 1px;
```

Comment:
Maybe a bit nit-picky but we have code in the [label](https://github.com/cds-snc/gcds-components/blob/main/packages/web/src/components/gcds-label/gcds-label.css#L9) component + the UF to hide text - can we use the same code here to be consistent?

```
  display: block;
  width: 0;
  height: 0;
  margin: 0;
  overflow: hidden;
```